mysite/account/tests.py

```diff
@@ -451,3 +451,27 @@ def test(self):
 
         self.assertTrue(self.user_logged_in(client1.session))
         self.assertFalse(self.user_logged_in(client2.session))
+
+class UrlRedirect(WebTest):
+    fixtures = ['user-paulproteus']
+
+    def test_appended_url_redirect(self):
+        client = Client()
```

Comment:
If you can add a docstring for the test function, that would be helpful for the next person working on the code.